lisp/pdf-view.el

```diff
@@ -405,6 +412,17 @@ PNG images in Emacs buffers."
   ;; Setup initial page and start display
   (pdf-view-goto-page (or (pdf-view-current-page) 1)))
 
+(when (fboundp 'advice-add)
+  (advice-add 'cua-copy-region
+	      :before-until
+	      #'cua-copy-region--pdf-view-advice))
+
+(defun cua-copy-region--pdf-view-advice (&rest _)
+  "If current buffer is in `pdf-view' mode, call
+`pdf-view-kill-ring-save'."
+  (when (eq major-mode 'pdf-view-mode)
+    (pdf-view-kill-ring-save)))
+
```

Comment:
You might as well define the function conditionally.

Reply:
Done.